core/modules/node/node.module

```diff
@@ -1863,6 +1865,10 @@ function node_block_info() {
     'description' => t('Displays a node in a block.'),
     'class' => 'NodeBlock',
   );
+  $blocks['empty'] = array(
+    'info' => t('Create content'),
+    'description' => t('A prompt to create some content with appropriate links.'),
+  );
```

Comment:
I really don't like this being a block. There would be zero usefulness of manually placing this block after the initial installation. A user would never place it if they knew what it was. For users that don't know what it is, it won't do anything at all.